strapi/strapi
# [V4] Prebuild the admin panel.

### What does it do?

It fixes the build script, now only the plugins that are shipped with a generated app will be built.
It also introduces a script that writes the plugins.js  file before starting the webpack-dev-server when developing in the monorepo.

### Why is it needed?

In order to only build the necessary plugin when releasing a new version of Strapi.

### How to test it?

The build script should build the admin correctly and the dev mode inside the monorepo should work as it used to.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
